tests/devapps/XForms/XForms/SettingsPage.xaml.cs

```diff
@@ -59,8 +59,7 @@ private void RefreshView()
             numOfAccountItems.Text = App.MsalPublicClient.UserTokenCacheInternal.Accessor.GetAllAccountsAsString()
                 .Count.ToString(CultureInfo.InvariantCulture);
 
-            // TODO(migration): how to handle this?
-            //validateAuthority.IsToggled = App.ValidateAuthority;
+            validateAuthority.IsToggled = App.ValidateAuthority;
```

Comment:
nit: it's more readable to suffix names of UI elements with what they are (redirectUriLabel, doStuffButton).

Reply:
I agree